cypress/integration/login.js

```diff
@@ -20,5 +20,6 @@ describe('Login functionality', function() {
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('Demo User')
         cy.get('.main-nav').find('li').should('have.length', 2)
+        cy.get('[data-test="timesheets-current-day"]').contains('15')
```

Comment:
1. You are trying to merge code to master and not your own develop branch.
2. This test will fail tomorrow, maybe we can do something better? e.g. a hint if you'll get stuck https://www.w3schools.com/jsref/jsref_getday.asp
3. data-test="timesheets-current-day" - is this locator unique?